src/main/java/com/lewisd/maven/lint/rules/AbstractRule.java

```diff
@@ -9,49 +9,67 @@
 
 public abstract class AbstractRule implements Rule {
 
-	protected static final String VERSION_PROPERTIES = "versionProperties";
-	protected static final String MAVEN_PROJECT = "mavenProject";
+  protected static final String VERSION_PROPERTIES = "versionProperties";

```

Comment:
This class looks like it's been reformatted with two-space indents instead of four-space like the rest of the codebase.  Was that intentional, or did something go wrong?